src/components/activities/Activity.jsx

```diff
@@ -137,21 +133,38 @@ const FullSizeCard = styled(Card)`
   overflow: auto;
 `;
 
-const ButtonWrapper = styled.div`
-  display: flex;
-
-  .button {
-    flex: 1;
-  }
-`;
-
 const TurnRevealWrapper = styled.div`
   .turnreveal-container {
     display: flex;
     justify-content: center;
   }
 `;
 
+const FlexContainer = styled.div`
```

Comment:
I thought we decided to try do these "wrapper" elements as little as possible. That was going relatively well in this component, but not anymore like this.

Reply:
I know, but this way I could combine
``` .title .location .price .button {
    flex-shrink: 1;
}```
into a single element, where otherwise I thought 4 different styled-components would be needed  to get the same effect, which would lead to a lot less readable code to me... But since this fel-shrink isn't necessary anymore, I reverted back to a styled-component.

Reply:
Ah I understand now. Fair enough :)